application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -373,7 +373,12 @@ public FeatureBlacklistConfig getFeatureBlacklistConfig() {
         return featureBlacklistConfig;
     }
 
-    public String getSelectRolesChannelPatten() {
-        return selectRolesChannelPatten;
+    /**
+     * Gets the REGEX pattern used to identify the role selection channel.
```

Comment:
try to use javadoc to elaborate on what the code doesnt directly explain already. for example:

> ... to identify the channel in which users can select their helper roles.